platform/mellanox/mlnx-sai/Makefile

```diff
@@ -7,6 +7,8 @@ DERIVED_TARGETS = mlnx-sai-dbgsym_1.mlnx.$(MLNX_SAI_VERSION)_$(CONFIGURED_ARCH).
 
 $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	pushd SAI-Implementation
```

Comment:
Do we need this directory? It's empty now, and the only use it is to store the downloaded mlnx_sai temporarily.